src/Generator.Bind/LicenseResource.cs

```diff
@@ -0,0 +1,32 @@
+using System.IO;
+using System.Reflection;
+
+namespace Bind
+{
+    /// <summary>
+    /// Class for handling embedded license text.
+    /// </summary>
+    internal class LicenseResource
+    {
+        private static string _license;
+
+        /// <summary>
+        /// Gets and returns the license text from the embedded resource.
```

Comment:
No need for "Gets and returns" - "Gets" is sufficient.

Reply:
Will fix it.